samples/bluetooth/hci_uart/sample.yaml

```diff
@@ -3,13 +3,12 @@ sample:
   name: TBD
 tests:
   sample.bluetooth.hci_uart.arm:
+    min_ram: 17
```

Comment:
I fixed this recently so it'd fit in 16KB of RAM. I'd really like to keep the bbc:microbit in the list of tested boards. Can you reduce the number of connections instead? See https://github.com/zephyrproject-rtos/zephyr/commit/9be6fb0ba275b62287632591374e35eb279b5a96